yarn.lock

```diff
@@ -37,6 +37,11 @@
   resolved "https://registry.yarnpkg.com/@agoric/nat/-/nat-2.0.1.tgz#28aba18367deba354bdd424bdf6daa48f5e7d37f"
   integrity sha512-puvWkoaJovQib/YaSRSGJ8Kn9rogi/yyaVa3d5znSZb5WiYfUiEKW35BfexHhAdi0VfPz2anFHoRBoBSUIijNA==
 
+"@agoric/nat@^3.0.1":
```

Comment:
this seems reasonable, but let's do a `yarn install` at the top (after deleting all the `yarn.lock` files from the subdirectories) and see if anything else changes.

Also, do a `yarn workspaces info |grep mismatched` and make sure none of the intra-monorepo dependencies are pointing to the wrong place. That will require having matching version numbers for everything.

Reply:
`yarn workspaces info |grep mismatched` is clean with no mismatched